content/project/projects/content+pt.lr

```diff
@@ -0,0 +1,14 @@
+_model: page
+---
+_template: projects.html
+---
+title: Projetos
+---
+summary: O que faz o projeto BeeWare?
+---
+sort_key: 4
+---
+body: BeeWare não é uma simples ferramenta ou biblioteca  - é uma coleção de projetos que pode ser usado para ajudar desenvolvedores, depurar e executar software Python. Cada ferramenta segue a filosofia Unix de fazer uma coisa correta. Cada ferramenta pode ser usada de forma isolada, 
```

Comment:
This was already translated, it's not showing the conflict because the file name is incorrect, it should've been `contents+pt.lr`.